docs: Made some edits (#1649)
EddieHubCommunity/support: README.md

### Some vague idea of how we could turn this into a support channel.

> Imagine the scenario where you are helping a stranger, a friend or a colleague on Discord / MS Teams / Slack / YouTube, etc. They may have found an old article from 5+ years ago about Docker that is not supported or a Node.js article that isn't supported in Node v14. That sort of thing...

Reviewer: ```suggestion
Imagine the scenario where you are helping a stranger, a friend, or a colleague on Discord / MS Teams / Slack / YouTube, etc. They may have found an old article from 5+ years ago about Docker that is not supported or a Node.js article that isn't supported in Node v14. That sort of thing...
```

Oxford commas are necessary

> You volunteer to help that person and pair on it. You fix it. They're super happy.

Reviewer 2: I think this one's better?:
`You volunteer to help that person and pair on it. You fix it and that made them super happy.`

Author: >You volunteer to help that person and pair on it. You fix it and that made them super happy.

This looks good. But, how about changing "made" to "make" to maintain the present tense being used?

Reviewer 2: Yeah better use "make"

make = present
made = past

Author: Making the change now

> You volunteer to help that person and pair on it. You fix it and that makes them super happy.

Reviewer: ```suggestion
You volunteer to help that person and pair on it. You fix it and they get super happy.
```

Might be economical?

- **What is a GitHub Organization?** Essentially it's a way to group many repositories in an entity called an organization. An organization also allows us to group people who contribute and are a part of the community. Some examples would be: [Netflix](https://github.com/Netflix), [Facebook](https://github.com/facebook) and of course [EddieHub](https://github.com/EddieHubCommunity).
Learn more on [GitHub docs](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/about-organizations).
- **How do I join the Github Organization?** Raise an issue on this repository asking to join ([invite me](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization)).

> - **How do I set the Organization to Public?** Here's how to make the organization public: 

Reviewer: Is this one really necessary? Rethink.

Author: @adityaraute The later sounds more direct. The listed items beneath would be able to start with needing to add "-ing"

> - **How do I set the Organization to Public?** Here's how to make the organization public: 

Reviewer 3: ```suggestion
- **How do I set the Organization to Public?** Here's how to publicize your organization membership:
```

Reviewer 3: Maybe this will be better 🆙

Reviewer 4: Why not deleting the whole second sentence?
It's really redundant information why writing it two times :)
```suggestion
- **How do I set the Organization to Public?**
```

Reviewer 3: I like your suggestion better, but I think that sentence must be there for some reason?

Reviewer 4: I could not think of any reason why we should keep this sentence 🙈

Reviewer: agree with @schmelto

	
- **How do I keep my Git fork up to date?** Add a new remote called `upstream` targeting the original repository. Check out more information on [this issue](https://github.com/EddieJaoudeCommunity/support/issues/94).
- **What is DevRel?** Check out the discussion on [this issue](https://github.com/EddieJaoudeCommunity/support/issues/64) or on [this YouTube video](https://www.youtube.com/watch?v=iUZerHctTB8&t=1534s).

> - **How do I get started with open source?** You can make your first pull request in this [repository](https://github.com/EddieJaoudeCommunity/hacktoberfest-practice). Follow the instructions in the README, and don't forget contributing to open source is not just about code, it's about collaboration, communication, and adding value. Don't be afraid to open an issue with a question :smiley:. If you have any trouble, you can ask questions in our Discord. We have a channel for `first-timers` and `opensource`.

Reviewer: ```suggestion
- **How do I get started with open source?** You can make your first pull request in this [repository](https://github.com/EddieJaoudeCommunity/hacktoberfest-practice). Follow the instructions in the README, and don't forget – contributing to open source is not just about code – it's about collaboration, communication, and adding value. Don't be afraid to open an issue with a question :smiley:. If you have any trouble, you can ask questions in our Discord. We have a channel for `first-timers` and `opensource`.
```

How about this structure instead?
